BUILD

```diff
@@ -116,6 +129,7 @@ tfrt_cc_library(
 tfrt_cc_binary(
     name = "clink_kernels_jna",
     srcs = [
+        "lib/feature/one_hot_encoder_jna.cc",
```

Comment:
For now can we move the methods from `one_hot_encoder_jna.cc` to `clink_kernels_jna.cc` for simplicity? I am not sure we need to define a new `xxx_jna.cc` file for every algorithms we implement in the future.

Reply:
OK. I'll make the change.